app/src/main/java/net/squanchy/tweets/service/TweetModelConverter.java

```diff
@@ -120,6 +114,25 @@ private int offsetFrom(int start, List<Integer> indices) {
         return Lists.map(photos, mediaEntity -> mediaEntity.mediaUrlHttps);
     }
 
+    private String displayableTextFor(Tweet tweet, Range displayTextRange, List<MediaEntity> photoUrls) {
+        Integer beginIndex = displayTextRange.start();
+        Integer endIndex = displayTextRange.end();
+        String displayableText = tweet.text.substring(beginIndex, endIndex);
+        if (photoUrls != null) {
```

Comment:
Isn't this from the Tweet class from the Twitter SDK? In that case I think it's gonna be null if there isn't any media attached